src/search/merge_and_shrink/merge_and_shrink_algorithm.cc

```diff
@@ -84,6 +84,9 @@ void MergeAndShrinkAlgorithm::dump_options() const {
                      << shrink_threshold_before_merge << endl;
         utils::g_log << endl;
 
+        shrink_strategy->dump_options();
```

Comment:
Is this output short enough to remain here long-term?

Reply:
Yes. The whole method here is also just `dump_options`, which is done once for M&S. (It used to be there but I think was lost when refactoring in an earlier issue.)